src/components/TodoList.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from 'react';
+import { TodosContext } from './TodosContext';
+import { TodoItem } from './TodoItem';
+
+export const TodoList: React.FC = () => {
+  const {
+    todos,
+    todosFilter,
+  } = useContext(TodosContext);
+
+  return (
+    <ul className="todo-list" data-cy="todosList">
+      {todos.filter((todo) => {
+        switch (todosFilter) {
+          case 'Active':
```

Comment:
Use a enum for these words
<img width="440" alt="image" src="https://github.com/mate-academy/react_todo-app/assets/64750958/b50ed79b-b501-4858-bdd0-7d811682e9f7">